lib/processors/bundlers/moduleBundler.js

```diff
@@ -46,7 +63,12 @@ class LocatorResourcePool extends ResourcePool {
  * @param {module:@ui5/fs.Resource[]} parameters.resources Resources
  * @param {Object} parameters.options Options
  * @param {Object} parameters.options.bundleDefinition Module bundle definition
+ * @param {string} parameters.options.bundleDefinition.name The module bundle name
+ * @param {string[]} parameters.options.bundleDefinition.defaultFileTypes List of default file types to be included in the bundle
+ * @param {ModuleSection[]} parameters.options.bundleDefinition.sections List of module sections.
  * @param {Object} parameters.options.bundleOptions Module bundle options
+ * @param {boolean} parameters.options.bundleOptions.optimize=false If set to 'true' the module bundles get minified
+ * @param {boolean} parameters.options.bundleOptions.usePredefineCalls=false If set to 'true', sap.ui.predefine is used for AMD compliant modules
```

Comment:
maybe better say "UI5 modules" instead of "AMD compliant modules" as we only rewrite sap.ui.define calls, not standard define calls.

Reply:
done